cedar-policy-validator/src/entity_manifest.rs

```diff
@@ -0,0 +1,1306 @@
+//! Entity Manifest definition and static analysis.
+
+/*
+ * Copyright Cedar Contributors
```

Comment:
In our other files, the copyright is at the top and any top-level comments come later. Not sure if there is an important reason for this, but might as well be consistent 🙂

Reply:
done (: